lib/core-integration/src/Test/Integration/Scenario/API/Wallets.hs

```diff
@@ -177,6 +177,37 @@ spec = do
             , expectFieldNotEqual passphraseLastUpdate Nothing
             ]
 
+    it "CREATE_WALLET_01 - SQL injection attempt when creating a wallet" $ \ctx -> do
+        let mnemonics = [ "pulp", "ten", "light", "rhythm", "replace"
+                        , "vessel", "slow", "drift", "kingdom", "amazing"
+                        , "negative", "join", "auction", "ugly", "symptom"] :: [Text]
+        let payload = Json [json| {
+                "name": "new wallet\",'',''); DROP TABLE \"wallet\"; --",
+                "mnemonic_sentence": #{mnemonics},
+                "passphrase": "12345678910"
+                } |]
+        r <- request @ApiWallet ctx (Link.postWallet @'Shelley) Default payload
+        verify r
+            [ expectResponseCode @IO HTTP.status201
+            , expectFieldEqual walletName "new wallet\",'',''); DROP TABLE \"wallet\"; --"
+            , expectFieldEqual
+                (#addressPoolGap . #getApiT . #getAddressPoolGap) 20
+            , expectFieldEqual (#balance . #getApiT . #available) (Quantity 0)
+            , expectFieldEqual (#balance . #getApiT . #total) (Quantity 0)
+            , expectFieldEqual (#balance . #getApiT . #reward) (Quantity 0)
+            , expectEventually ctx (Link.getWallet @'Shelley)
+                (#state . #getApiT) Ready
+            , expectFieldEqual delegation (NotDelegating)
+            , expectFieldEqual walletId
+                "135bfb99b9f7a0c702bf8c658cc0d9b1a0d797a2"
+            , expectFieldNotEqual passphraseLastUpdate Nothing
+            ]
+        rl <- request @[ApiWallet] ctx (Link.listWallets @'Shelley) Default Empty
```

Comment:
checking if the table is not dropped although we are sure because of the above : 
```
expectFieldEqual walletName "new wallet\",'',''); DROP TABLE \"wallet\"; --"
```